sample/build.gradle

```diff
@@ -162,13 +63,13 @@ android {
 
 dependencies { configuration ->
 
-    //releaseTestKujakuImport(configuration)
-    developmentKujakuModulesImport(this, configuration)
+    //libraryModuleRelease(configuration)
+    libraryModuleDevelopment(this, configuration)
```

Comment:
We need some documentation on the usage of these _i.e. release vs dev mode_ e.g. on the READ ME

Reply:
> The PR looks good for merge, however I think we should create a ticket around reporting the Instrumentation coverage. cc @ekigamba

I agree. Here is the ticket https://github.com/onaio/kujaku/issues/358

Reply:
An issue to update the documentation here https://github.com/onaio/kujaku/issues/359